src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, Calculate {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public Figure() {
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public double getArea() {
+        // метод расчета площади фигуры
+        return 0.0;
+    }
+
+    public void draw() {
+        // метод отображения информации о фигуре
```

Comment:
don't forget to remove the comments before committing

Reply:
remove comment